### PR TITLE
chore(sidekick): prepare for manifest v3 version

### PR DESCRIPTION
**Pull request by rofe**

### CODE REVIEW

File: tools/sidekick/plugins.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    const resp = await fetch(purgeUrl, { method: 'PURGE' });
+    if (!resp.ok) {
+      throw new Error(`non-ok status ${resp.status}`);
+    }
+    // refresh browser cache
+    await fetch(purgeUrl, { cache: 'reload' });
+  } catch (e) {
+    console.error(`failed to purge ${purgeUrl}: ${e.message}`);
+  }
+};
```
Comment (rofe) [inline tools/sidekick/plugins.js:33]: @davidnuescheler do you know if this sidekick extension is still needed? It looks like something from pre-hlx3 times...

Comment (rofe) [reply tools/sidekick/plugins.js:33]: I'll remove it for now.

**Comment by amauch-adobe** [comment]
I'm assuming for testing this PR these would be the instructions.  I'm pulling these comments from your PR in the FedPub repo.

https://sidekick-mv3--fedpub--adobe.hlx.page/creativecloud/photography/hub/guides/guide-to-aperture-vs-iris

To try it out, create a local checkout of the [helix-sidekick-extension](https://github.com/adobe/helix-sidekick-extension) repo, switch to the mv3 branch and follow these instructions:
https://github.com/adobe/helix-sidekick-extension#local-testing

cc @rofe

**Review by amauch-adobe — CHANGES_REQUESTED** [review]
@rofe, I followed your instructions for testing this out, meaning I installed the mv3 version of the helix sidekick by using the branch specified. I then added the Pages (PACOM) site to the helix sidekick extension and then went to this URL (https://pages.adobe.com/lightroom/en/wg/lightroom-mobile-feedback) for testing. When I did that I no longer could see the different plugin buttons specific to the PACOM config.json in your PR, meaning on the preview view the "Parent Folder" button was no longer there and also was gone when in edit mode/view. I noticed the "Parent Folder" was a specific config option according to the JSON so I was wondering why it wasn't showing up for the helix sidekick mv3 version?

Other than that concern, everything else looked good.

**Comment by rofe** [comment]
@amauch-adobe is it working on https://main--pages--adobe.hlx.page/lightroom/en/wg/lightroom-mobile-feedback?

Note that the parent folder plugin is currently configured to only show up in the `dev` and `preview` environments. If you need it for `live` and/or `prod` as well, you can add them to the `environments` array in `config.json`.

**Comment by rofe** [comment]
Sorry, this can of course only be tested on the `sidekick-mv3` branch of this project, so you would need to set your sidekick to use `https://github.com/adobe/pages/tree/sidekick-mv3` as GitHub URL and then come here:

https://sidekick-mv3--pages--adobe.hlx.page/lightroom/en/wg/lightroom-mobile-feedback

**Comment by rofe** [comment]
@amauch-adobe have you been able to test this with the updated instructions above?

**Review by amauch-adobe — APPROVED** [review]
Everyone looks good now.

**Pull request merged**